shop/models/fields.py

```diff
@@ -48,9 +48,13 @@ def __call__(cls, value, *args, **kwargs):
                 pass  # let the super method complain
         return super(ChoiceEnumMeta, cls).__call__(value, *args, **kwargs)
 
+if PY2:
```

Comment:
For better readability, I would prefer to use ``six.with_metaclass`` as a compatibility layer, or is there any special reason not to use it?

Reply:
By using six as this issue, there is this error:
  File "/home/np/sources/Tutorial/shoptutorial/lib/python3.6/enum.py", line 135, in __new__
    enum_members = {k: classdict[k] for k in classdict._member_names}
AttributeError: 'dict' object has no attribute '_member_names'